mmv1/products/redis/Cluster.yaml

```diff
@@ -231,6 +231,14 @@ properties:
     description: |
       Required. Number of shards for the Redis cluster.
     required: true
+  - !ruby/object:Api::Type::Boolean
+    name: deletionProtectionEnabled
+    description: |
+      Optional. Indicates if the cluster is deletion protected or not.
+      If the value if set to true, any delete cluster operation will fail.
+      Default value is true.
+    required: false
+    default_value: true
```

Comment:
I think this might be a better fit for `default_from_api` if this is also a server-side default -- if this config was manually overridden for a pre-existing resource by a user via cloud console or pantheon, adding a field with a default value could potentially introduce a breaking change for users' configs (i.e. it potentially planning to do `false -> true` for users who don't manually add `deletion_protection_enabled = false` to their config)

Reply:
Hey @NickElliot , it is not the default value. The server-side default is false. It will also not be backward compatible for existing terraform plans. Is my understanding correct?